test/e2e/handler/node_selector_test.go

```diff
@@ -81,6 +84,8 @@ var _ = Describe("[rfe_id:3503][crit:medium][vendor:cnv-qe@redhat.com][level:com
 						Type:   nmstate.NodeNetworkConfigurationEnactmentConditionMatching,
 						Status: corev1.ConditionTrue,
 					}))
+				//TODO: Remove this when webhook retest policy status when node labels are changed
+				time.Sleep(3 * time.Second)
```

Comment:
is this to make sure we don't hit the available state from before adding the label?

Reply:
You got it, I am implementing the webhook to prevent that, for now we need this Sleep.